libcxx/test/std/utilities/variant/variant.variant/variant.assign/conv.pass.cpp

```diff
@@ -33,8 +33,13 @@ int main(int, char**)
 
   static_assert(!std::is_assignable<std::variant<int, bool>, decltype("meow")>::value, "");
   static_assert(!std::is_assignable<std::variant<int, const bool>, decltype("meow")>::value, "");
-
+#if defined(_LIBCPP_ENABLE_NARROWING_CONVERSIONS_IN_VARIANT)
```

Comment:
nit: it seems like `#ifdef` is preferred over `#if defined(...)`, I'd go w/ that.